payloads/v9/invite.ts

```diff
@@ -3,12 +3,33 @@
  */
 
 import type { APIPartialChannel } from './channel';
-import type { APIPartialGuild } from './guild';
+import type { APIGuild } from './guild';
 import type { APIApplication } from './application';
 import type { APIInviteStageInstance } from './stageInstance';
 import type { APIUser } from './user';
 import type { APIGuildScheduledEvent } from './guildScheduledEvent';
 
+export interface APIInviteGuild
+	extends Pick<
+		APIGuild,
+		| 'id'
+		| 'name'
+		| 'splash'
+		| 'banner'
```

Comment:
If these fields (and icon and vanity url are always null, don't pick them from the base guild interface